ebean-migration/src/main/java/io/ebean/migration/MigrationConfig.java

```diff
@@ -418,17 +423,34 @@ public void setClassLoader(ClassLoader classLoader) {
   /**
    * Return the jdbcMigrationFactory.
    */
-  public JdbcMigrationFactory getJdbcMigrationFactory() {
+  public Iterable<JdbcMigration> getJdbcMigrationFactory() {
```

Comment:
We have a name mismatch in that `getJdbcMigrationFactory()` returns a "collection of migrations".  

So I get it but it seems off. Migrations need to be sorted into version order etc. This doesn't quite look right yet.

Reply:
agree, I'll rename it to `getJdbcMigrationCollection`

> Migrations need to be sorted into version order etc. This doesn't quite look right yet.

I don't get you here. They are sorted later here https://github.com/ebean-orm/ebean-migration/pull/151/files#diff-10ad52c1950c7a3f96d7f7286cee908ff73e79324b8414c53ce3ad17204ca9e1R57 and they have to be sorted between the SQL migrations